src/core/vector/qgsvectorlayereditbuffergroup.h

```diff
@@ -91,14 +92,14 @@ class CORE_EXPORT QgsVectorLayerEditBufferGroup : public QObject
      *
      * \returns TRUE on success
      */
-    bool commitChanges( QStringList &commitErrors, bool stopEditing = true );
+    bool commitChanges( QStringList &commitErrors SIP_OUT, bool stopEditing = true );
```

Comment:
If you add "\param commitErrors will be set to a list of descriptive error messages if the commit fails" and "\param stopEditing .... etc" here, then sipify will properly handle SIP_OUT when generating Python docstrings. (same with rollback)